softdevice_controller/CHANGELOG.rst

```diff
@@ -87,11 +110,11 @@ Bug fixes
 =========
 
 * Fixed an issue where the application could not immediately restart a connectable advertiser after a high duty cycle advertiser timed out (DRGN-13029).
-* Fixed an issue where a directed advertiser used a resolvable address as the TargetA when the local device address was set to public or random device address (DRGN-13921).
-* Fixed an issue where "HCI LE Set Extended Advertising Parameters" should have returned "Packet Too Long (0x45)" when the advertising set was already configured with data which was longer than it could fit within the advertising interval.
+* Fixed an issue where a directed advertiser used a resolvable address as the target when the local device address was set to public or random device address (DRGN-13921).
```

Comment:
TargetA was intentionally used, it is the name of a PDU field.
```suggestion
* Fixed an issue where a directed advertiser used a resolvable address as the ``TargetA`` when the local device address was set to public or random device address (DRGN-13921).
```

Reply:
fixed